webapp/packages/plugin-connections/src/ConnectionAuthentication/NetworkHandlerAuthForm.tsx

```diff
@@ -56,7 +56,7 @@ export const NetworkHandlerAuthForm = observer<Props>(function NetworkHandlerAut
           <InputField type="text" name="userName" state={state} disabled={disabled}>
             {translate(`connections_network_handler_${id}_user`, 'connections_network_handler_default_user')}
           </InputField>
-          <InputField type="password" name="password" canShowPassword={false} state={state} disabled={disabled}>
+          <InputField autoComplete="new-password" type="password" name="password" canShowPassword={false} state={state} disabled={disabled}>
```

Comment:
it's better to keep same main props order for better readability (like type, name, state, disabled ...)